core/kamon-core/src/main/resources/reference.conf

```diff
@@ -176,6 +176,10 @@ kamon {
     # as failed.
     include-error-stacktrace = yes
 
+    # Decides whether to include the class name of a Throwable as the "error.type" Span tag when a Span is marked
+    # as failed.
+    include-error-type = no
```

Comment:
Why not have it on by default? It feels like a good thing to have for filtering failed spans